certgen/csr_cryptocb.c

```diff
@@ -44,6 +44,18 @@
 #define ENABLE_CSR_EXAMPLE
 #endif
 
+/* Private and public key files for signing */
+#define ECC_KEY_FILE        "../certs/ecc-key.pem"
+#define ECC_KEYPUB_FILE     "../certs/ecc-keyPub.pem"
+#define RSA_KEY_FILE        "../certs/client-key.pem"
+#define RSA_KEYPUB_FILE     "../certs/client-keyPub.pem"
+#define ED25519_KEY_FILE    "../certs/ed25519-keyPriv.pem"
+#define ED25519_KEYPUB_FILE "../certs/ed25519-keyPub.pem"
+
+/* openssl ec -inform pem -in ecc-key.pem -outform pem -out ecc-keyPub.pem -pubout */
```

Comment:
Might want to say "These public key files were generated using the following commands"